packages/design-system/src/components/flex.tsx

```diff
@@ -90,6 +92,22 @@ export const Flex = styled("div", {
         gap: "$spacing$20",
       },
     },
+    shrink: {
+      true: {
+        flexShrink: 1,
+      },
+      false: {
+        flexShrink: 0,
+      },
```

Comment:
What happens if you don't pass `shrink` at all, neither `false` nor `true`, just `<Flex>`?

- If neither `false` nor `true` is applied, then we're good. 
- If `false` is applied, we potentially break usage of `<Flex>` somewhere as we're setting `flexShrink:0` for all `Flex`es by default.

Reply:
neither is applied